static/app/views/detectors/new-settings.tsx

```diff
@@ -1,44 +1,107 @@
+import {useFormField} from 'sentry/components/workflowEngine/form/useFormField';
 import {
   StickyFooter,
   StickyFooterLabel,
 } from 'sentry/components/workflowEngine/ui/footer';
 import {useWorkflowEngineFeatureGate} from 'sentry/components/workflowEngine/useWorkflowEngineFeatureGate';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
-import type {Detector} from 'sentry/types/workflowEngine/detectors';
+import {useLocation} from 'sentry/utils/useLocation';
 import {useNavigate} from 'sentry/utils/useNavigate';
 import useOrganization from 'sentry/utils/useOrganization';
+import {EditableDetectorName} from 'sentry/views/detectors/components/forms/editableDetectorName';
 import {MetricDetectorForm} from 'sentry/views/detectors/components/forms/metric';
+import type {MetricDetectorFormData} from 'sentry/views/detectors/components/forms/metricFormData';
+import {
+  DEFAULT_THRESHOLD_METRIC_FORM_DATA,
+  getNewMetricDetectorData,
+} from 'sentry/views/detectors/components/forms/metricFormData';
 import {useCreateDetector} from 'sentry/views/detectors/hooks';
-import NewDetectorLayout from 'sentry/views/detectors/layouts/new';
 import {
   makeMonitorBasePathname,
   makeMonitorDetailsPathname,
 } from 'sentry/views/detectors/pathnames';
 
+function NewDetectorBreadcrumbs() {
+  const title = useFormField<string>('title');
+  const organization = useOrganization();
+  return (
+    <Breadcrumbs
+      crumbs={[
+        {label: t('Monitors'), to: makeMonitorBasePathname(organization.slug)},
+        {label: title ? title : t('New Monitor')},
+      ]}
+    />
+  );
+}
+
+function NewDetectorDocumentTitle() {
+  const title = useFormField<string>('title');
+  return <SentryDocumentTitle title={title ? title : t('New Monitor')} />;
+}
+
 export default function DetectorNewSettings() {
+  const location = useLocation();
   const organization = useOrganization();
   useWorkflowEngineFeatureGate({redirect: true});
   const navigate = useNavigate();
-  const model = useMemo(() => new FormModel(), []);
 
   const {mutateAsync: createDetector} = useCreateDetector();
 
-  const handleSubmit = useCallback(async () => {
-    const data = model.getData() as unknown as Detector;
-    const result = await createDetector(data);
-    navigate(makeMonitorDetailsPathname(organization.slug, result.id));
-  }, [createDetector, model, navigate, organization]);
+  const handleSubmit = useCallback<OnSubmitCallback>(
+    async (data, _, __, ___, formModel) => {
+      const hasErrors = formModel.validateForm();
+      if (!hasErrors) {
+        return;
+      }
+
+      const detector = await createDetector(
+        getNewMetricDetectorData(data as MetricDetectorFormData)
+      );
+      navigate(makeMonitorDetailsPathname(organization.slug, detector.id));
+    },
+    [createDetector, navigate, organization.slug]
+  );
 
   return (
-    <NewDetectorLayout>
-      <MetricDetectorForm model={model} />
+    <FullHeightForm
```

Comment:
is any of this stuff something we should make shared components from?

It seems like we could have a lot of little split out components to me, 
```
        <StyledLayoutHeader>
          <Layout.HeaderContent>
            <NewDetectorBreadcrumbs />
            <Layout.Title>
              <EditableDetectorName />
            </Layout.Title>
          </Layout.HeaderContent>
        </StyledLayoutHeader>
```
might be a shareable `DetectorHeader` -- need to find those common components because we'll have so many different detector types. (the other trick is to make them composable to allow for overrides)